cores/arduino/USBSerial.cpp

```diff
@@ -185,6 +187,11 @@ bool USBSerial::rts(void)
   return false;
 }
 
+bool USBSerial::isConnected()
```

Comment:
Sorry, could you remove it as in fact deprecated. bool operator is the right way.